### ISSUE TITLE
gr.get.params example crashes BASIC!

### ISSUE THREAD

**Issue by mougino** [RFO-BASIC/Basic#192]
See http://rfobasic.freeforums.org/post22368.html#p22368
Code that crashes BASIC! is
```VB
gr.open
gr.text.draw t,10,10,"bla bla"
gr.get.params t,p$[]
gr.close
array.size as,p$[]
for i=1 to as: ?p$[i]: next
do: pause 1: until 0
```

**Comment by jMarcS** [comment]
There's an odd bit of code in `execute_gr_get_params()` that tries to cast the `List` return value of `Arrays.asList` to `ArrayList`, which is clearly does not work. It went in back in March with a whole lot of unrelated GR changes, so it's been broken since v01.86. Well, it's fixed now. Not committed yet, but will be soon.

**Comment by jMarcS** [comment]
Released in v01.88, 2015/08/16.
